Gemfile

```diff
@@ -7,7 +7,7 @@ group :test do
   gem 'voxpupuli-test', '~> 7.0',   :require => false
   gem 'coveralls',                  :require => false
   gem 'simplecov-console',          :require => false
-  gem 'puppet_metadata', '~> 3.5',  :require => false
+  gem 'puppet_metadata', github: 'voxpupuli/puppet_metadata', :require => false
```

Comment:
@bastelfreak if we can cut a release of the puppet metadata, this can go back to the gem version

Reply:
https://rubygems.org/gems/puppet_metadata/versions/4.1.0

Reply:
Done!